Description: rename local sqlite3_log that is now included in libsqlite3
Author: Sean Bright <sean@malleable.com>
Bug: https://issues.asterisk.org/view.php?id=17017
Origin: http://svn.digium.com/svn/asterisk/branches/1.6.2@252317 

Fix building CDR and SQLite3 modules.

They added a sqlite3_log() function which was conflicting with our function
names.

diff --git a/cdr/cdr_sqlite3_custom.c b/cdr/cdr_sqlite3_custom.c
index a16648b..b1d317c 100644
--- a/cdr/cdr_sqlite3_custom.c
+++ b/cdr/cdr_sqlite3_custom.c
@@ -220,7 +220,7 @@ static void free_config(int reload)
 	}
 }
 
-static int sqlite3_log(struct ast_cdr *cdr)
+static int write_cdr(struct ast_cdr *cdr)
 {
 	int res = 0;
 	char *error = NULL;
@@ -321,7 +321,7 @@ static int load_module(void)
 		}
 	}
 
-	res = ast_cdr_register(name, desc, sqlite3_log);
+	res = ast_cdr_register(name, desc, write_cdr);
 	if (res) {
 		ast_log(LOG_ERROR, "Unable to register custom SQLite3 CDR handling\n");
 		free_config(0);
